lua/neogen/configurations/python.lua

```diff
@@ -247,8 +247,8 @@ return {
                         end
 
                         if nodes[i.Return] then
-                            validate_bare_returns(nodes)
                             validate_direct_returns(nodes, node)
+                            validate_bare_returns(nodes)
```

Comment:
This is from https://github.com/danymat/neogen/pull/171. Added it here so that more unittests could pass

Reply:
Already merged, so i'm resolving this